docs/api-reference.md

```diff
@@ -12,6 +12,7 @@ route: '/reference/api'
 - [`cleanup`](/reference/api#cleanup)
 - [`addCleanup`](/reference/api#addcleanup)
 - [`removeCleanup`](/reference/api#removecleanup)
+- [`console.error`](/reference/api#consoleerror)
```

Comment:
Just reading what this list of links is for, I don't this this one should be added here after all (it will appear in the side menu still though).